services/worker/src/worker/job_runners/split/descriptive_statistics.py

```diff
@@ -239,27 +244,39 @@ def compute_categorical_statistics(
     con: duckdb.DuckDBPyConnection,
     column_name: str,
     table_name: str,
-    class_label_names: list[str],
+    class_label_feature: ClassLabel,
     n_samples: int,
 ) -> CategoricalStatisticsItem:
     categorical_counts_query = COMPUTE_CATEGORIES_COUNTS_COMMAND.format(
         column_name=column_name, data_table_name=table_name
     )
     logging.debug(f"Compute categories counts for {column_name}.\n{categorical_counts_query}")
-    ids_to_counts: dict[int, int] = dict(
+    ids2counts: dict[int, int] = dict(
```

Comment:
nit: why changing this?

Reply:
for nothing actually, i think usually people do it with `2` (for example `datasets` has `ClassLabel.str2int()`, `ClassLabel.int2str()`)
can revert

Reply:
it's not important, but it diverts a bit while reviewing (as we try not to change the style/format of JSON files for example, I prefer when "form-only" changes are in their own PR)